FirebaseInstallations/Source/Library/FIRSecureStorage.h

```diff
@@ -20,18 +20,42 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/// The class provides a convenient abstraction on top of the iOS Keychain API to save data.
 @interface FIRSecureStorage : NSObject
 
 - (instancetype)initWithService:(NSString *)service;
 
+/** Get an object by key.
+ *  @param key The key.
+ *  @param objectClass The expected object class required by `NSSecureCoding`
+ *  @param accessGroup The Keychain Access Group.
+ *
+ *  @return Returns a promise. It is resolved with an object stored by key if exists. It is resolved
+ * with `nil` when the object not found. It fails on a Keychain error.
+ */
 - (FBLPromise<id<NSSecureCoding>> *)getObjectForKey:(NSString *)key
                                         objectClass:(Class)objectClass
                                         accessGroup:(nullable NSString *)accessGroup;
 
+/** Saves the given object by the given key.
+ *  @param object The object to store.
+ *  @param key The key to store the object. If there is an existing object by the key, it will be
+ * overriden.
+ *  @param accessGroup The Keychain Access Group.
+ *
+ *  @return Returns which is resolved with [NSNull null] on success.
```

Comment:
Nit: please add backticks around `[NSNull null]`

Reply:
Also, can anything fail here?

Reply:
Yes, any keychain errors will lead to the promise rejection with the error.